static/templates/dashboard/dashboard.html

```diff
@@ -187,4 +187,38 @@
     </div>
 
 </md-data-table>
+</div>
+
+  <md-sidenav md-component-id="right" md-is-locked-open="$mdMedia('min-width: 333px')" class="md-sidenav-right" flex="100">
+      <md-list flex="100">
+          <md-subheader class="md-no-sticky">
+            Rank Requesters<br /><br/>
+            <input type="text" ng-model="filterRequester" placeholder="filter requesters..."/>
```

Comment:
you can use an autocomplete here if you want which uses the region below as filteredResults (not the default dialog)